Finjector.Web/ClientApp/src/App.test.tsx

```diff
@@ -20,6 +20,69 @@ describe("App", () => {
         screen.getByText("New Chart String from Scratch")
       ).toBeInTheDocument();
     });
+
+    await waitFor(() => {
+      expect(screen.getByText("Finjector")).toBeInTheDocument();
+    });
+    await waitFor(() => {
+      expect(screen.getByText("Chart Strings")).toBeInTheDocument();
+    });
+    await waitFor(() => {
+      expect(screen.getByText("Teams")).toBeInTheDocument();
+    });
+    await waitFor(() => {
+      expect(screen.getByText("Help")).toBeInTheDocument();
+    });
+
+    //console.log(screen.debug());
+  });
+
+  it("Has link to the home page from Finjector", async () => {
+    render(wrappedView());
+
+    // should see landing page
+    await waitFor(() => {
+      expect(
+        screen.getByRole("link", { name: /Finjector/i })
+      ).toBeInTheDocument();
+      const link = screen.getByRole("link", { name: /Finjector/i });
+      expect(link).toHaveAttribute("href", "/");
+    });
```

Comment:
you can do this all just in one, like ```
await waitFor(() => {
      const link = screen.getByRole("link", { name: /Finjector/i });
      expect(link).toHaveAttribute("href", "/");
    });
    ```

if it isn't in the document it'll throw an exception so no need to check for that